src/lib.rs

```diff
@@ -19,6 +19,44 @@ const DEBUG: bool = false;
 
 type PluginHandle = u32;
 
+//START AE CODE
```

Comment:
Why not copy & paste code from dc7571b981d8d5d9eb86371a83fbe46bb164595f ?
I want to remove these comments, rename enum, insert spaces between u8; and 256, and apply zero_pad_u8.

Reply:
Well, you told me to rebase on to master which doesn't contain that commit, but of course I could do that.